packages/element-lib/build/contracts/Migrations.json

```diff
@@ -1379,12 +1379,12 @@
     "133700": {
```

Comment:
For testnet and mainnet, its nice to have contract addresses checked in, since we use a private network, we also see changes every time we run migrate... probably we can ignore the build folder, its not like contracts should be changing frequently.

Reply:
I have removed it from source, we will need to make sure it does get published to npm though.